rpcs3/Emu/Memory/vm.h

```diff
@@ -16,6 +16,7 @@ namespace vm
 	{
 		main,
 		user_space,
+		rsx_context,
```

Comment:
You don't need this. First 4 entries are special and they aren't required to allocate something with falloc.